SPECS-EXTENDED/libreswan/libreswan.signatures.json

```diff
@@ -0,0 +1,8 @@
+{
+ "Signatures": {
+  "libreswan-4.7.tar.gz": "ddd6337b3900063d870301c3d9f61f56107c765850fb00a163d360359ff3fc44",
+  "ikev1_dsa.fax.bz2": "030c7ac59422c2c36ed332efca925b18dd842da138619daab65c15663c687086",
+  "ikev1_psk.fax.bz2": "4698f0d8e653e20f279dd54aa1a270aadcc3e99da9e3a91852af12a644cc3531",
+  "ikev2.fax.bz2": "36f356538cdcab6a1712425ad386f32834f0dc333acbc4cc642db0f910be1d21"
```

Comment:
I think you accidentally checked-in these 3 binaries into the repo.

Reply:
fixed!